src/routes/mods.rs

```diff
@@ -79,6 +86,19 @@ pub async fn mod_get(
 
     if let Some(data) = mod_data {
         let m = data.inner;
+
+        let status = sqlx::query!(
+            "
+                SELECT status FROM statuses
+                WHERE id = $1
+                ",
+            m.status.0,
+        )
+        .fetch_one(&**pool)
+        .await
+        .map_err(|e| ApiError::DatabaseError(e.into()))?
+        .status
+        .unwrap_or(ModStatus::Unknown.to_string());
```

Comment:
Same as the above.  This could also be improved by using `unwrap_or_else`, but that would still be less efficient than just converting it to a `ModStatus` here.